lib/service/supervisor.go

```
@@ -197,7 +213,13 @@ func NewSupervisor(id string, parentLog *slog.Logger) Supervisor {
 		signalGracefulExit:  signalGracefulExit,
 
 		log: parentLog.With(teleport.ComponentKey, teleport.Component(teleport.ComponentProcess, id)),
+
+		clock: clock,
 	}
+
+	// used by processState
+	_ = metrics.RegisterPrometheusCollectors(stateGauge)
```

Is it ok to now silently drop the error? 🤔